frontend/public/locales/fr/apply-adult.json

```
@@ -47,18 +47,18 @@
     }
   },
   "confirm": {
-    "page-title": "(FR) Application successfully submitted",
+    "page-title": "Demande soumise avec succès",
     "app-code-is": "Votre code de demande est\u00a0:",
     "make-note": "Prenez ce code en note, car vous en aurez besoin pour vérifier l'état de votre demande.",
     "keep-copy": "Conservez une copie de votre demande",
-    "print-copy-text": "Vous <strong>ne recevrez pas</strong> de <strong>courriel de confirmation</strong>. Il est important de conserver une copie de votre demande. <noPrint>Vous pouvez utiliser ce bouton pour imprimer.<noPrint>",
+    "print-copy-text": "Vous ne recevrez <strong>pas</strong> de courriel de confirmation. Il est important de conserver une copie de votre demande. <noPrint>Vous pouvez utiliser ce bouton pour imprimer</noPrint>.",
```

`<noPrint>` will not render a link but will wrap that  content with `<span className="print:hidden" />`. The period `.` has to be inside the `<noPrint>`.

```suggestion
    "print-copy-text": "Vous ne recevrez <strong>pas</strong> de courriel de confirmation. Il est important de conserver une copie de votre demande. <noPrint>Vous pouvez utiliser ce bouton pour imprimer.</noPrint>",
```